### PR TITLE
Feat: Add homepage sections and documentation

### PR DESCRIPTION
**Pull request by nam-t24**
I just did this during class lol

You'll see added homepage sections underneath the hero section and documentation in DEMO.md for Prof and TA to refer to in order to run the project.

### CODE REVIEW

**Comment by blst15ki** [comment]
For the "How It Works" section, how are the borders supposed to be set up?

image
image

In the first image the border is on two sides, but in the second image the border is on three sides.

**Comment by nam-t24** [comment]
It's random by design

File: components/homepage/howItWorks.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          {/* Create organization section */}
+          <section className="flex items-center justify-between">
+            {/* Text */}
+            <div className="w-[30rem]">
+              <div className="2xl:text-4xl text-3xl font-semibold">
+                Create your <p className="text-maroon inline">organization</p>
+              </div>
+              <div className="text-gray-500 mt-4 text-lg">
+                Create your organization and invite your team members to join.
+                If you are your own organization, create an organization to
```
Comment (blst15ki) [inline components/homepage/howItWorks.tsx]: Weird to read create an organization twice essentially

File: components/homepage/howItWorks.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            <div className="w-[30rem]">
+              <div className="2xl:text-4xl text-3xl font-semibold">
+                Review <p className="text-maroon inline">applications</p> and
+                choose a <p className="text-maroon inline">recipient</p>
+              </div>
+              <div className="text-gray-500 mt-4 text-lg">
+                Review applications submitted by non-profits and choose a
+                recipient for your grant. After choosing a recipient you will be
+                able to create an appointment time to set up communications with
+                the non-profit organization on how to proceed. Non-profits can
```
Comment (blst15ki) [inline components/homepage/howItWorks.tsx]: Not sure if that's the direction we're going (creating appointments to communicate)

File: components/homepage/howItWorks.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        <div className="2xl:mt-28 mt-20">
+          <div className="2xl:text-5xl text-4xl font-semibold text-center 2xl:mb-8 mb-6">
+            Other <p className="text-maroon inline">features</p>
+          </div>
+          <div className="flex justify-center text-gray-500 text-lg">
+            <ul className="list-disc">
+              <li>Secure login authentication</li>
+              <li>Manage your user profile</li>
+              <li>
+                Manage your team by promoting, demoting, and kicking members
```
Comment (blst15ki) [inline components/homepage/howItWorks.tsx]: Maybe just "Manage your organization" instead

File: components/homepage/whyMinted.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      <div id="WhyMinted" className="w-full bg-maroon/20 text-center 2xl:pt-20 pt-16 overflow-hidden">
+        <div className="2xl:text-6xl text-5xl font-semibold text-darkmaroon 2xl:mb-10 mb-8">
+          Why Minted
+        </div>
+        <div className="text-gray-600 2xl:text-xl text-lg w-[55rem] mx-auto flex flex-col space-y-8">
+          <div>
+            Minted is a feature rich application that connects donors and
+            non-profits in a seamless and efficient way. This app is dedicated
+            to help boost community projects by helping non-profits get the
+            funding they need. All features are user friendly with an easy to
```
Comment (blst15ki) [inline components/homepage/whyMinted.tsx]: "feature-rich" instead of feature rich
"user-friendly" instead of user friendly (to match About content)

**Review by blst15ki — CHANGES_REQUESTED** [review]

File: components/homepage/howItWorks.tsx (hunk 1 above)
Comment (nam-t24) [reply components/homepage/howItWorks.tsx]: I'll do this tmr but what wording do you suggest

Comment (blst15ki) [reply components/homepage/howItWorks.tsx]: "Create your organization with your team to manage your grants or applications." or "Create your organization to add your team and manage your grants or applications."

Maybe replace the first 2 sentences with this

**Review by blst15ki — APPROVED** [review]

**Pull request merged**